lib/import/admin.js

```diff
@@ -142,17 +150,57 @@ function save_features(feature_collection) {
 }
 
 /**
+ * Get hash of admin to population
+ * mutates `feature_collection`.
```

Comment:
I don't quite follow this.

This could be related to: "...tease apart the enrich_features/save_topojson/save_features thing"